storages/jsonl-storage/src/store.rs

```diff
@@ -0,0 +1,65 @@
+use {
+    crate::error::{JsonlStorageError, OptionExt, ResultExt},
+    crate::JsonlStorage,
```

Comment:
nit. we can merge this in to a single `crate::{..}`.
this can be also applied to `store_mut.rs`